src/main/kotlin/no/nav/personbruker/dittnav/api/logging/logTokenExpiry.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.personbruker.dittnav.api.logging
+
+import io.ktor.application.ApplicationCall
+import io.ktor.response.ResponseHeaders
+import org.slf4j.Logger
+
+fun logTokenExpiry(log: Logger, call: ApplicationCall) {
+    val headers: ResponseHeaders = call.response.headers
+
+    if (headers.contains("x-token-expires-soon")) {
```

Comment:
Her tenker jeg at det er bedre å sjekke hvor lenge det faktisk er til token-et utløper, slik at dette kan gjøres uavhengig av hvilke headere som er satt.

Reply:
Hvorfor er det bedre hvis den samme logikken uansett ligger i Tokensupport? Har testet at dette fungerte i Q1 og alt så bra ut :)

Reply:
Fordi det ikke er det vi skal teste🙂  Jeg ønsker å se om vi får kall inn til `dittnav-api` med token som har kortere levetid enn X antall minutter, helt uavhengig av hvilke headere som er satt. Ønsker også å kunne sette X til noe annet enn det som Tokensupport bruker.

Reply:
Her ønsker vi å sjekke utløpstiden til et token som kommer i en request inn til `dittnav-api`, og det er ikke sikkert at `tokensupport` har vært inne i bildet tidligere for å sette den header-en. `Tokensupport` setter normalt sett denne header-en på responsen fra `dittnav-api`, og ikke på request-en den får fra frontend-en 🙂